api/ceramic.yaml

```diff
@@ -15,6 +15,60 @@ servers:
   - url: /ceramic
 
 paths:
+  /auth/token/refresh:
+    post:
+      summary: Create a refresh token
+      parameters:
+        - name: Authorization
+          in: header
+          description: Bearer token
+          required: true
+          schema:
+            type: string
+        - name: Did
```

Comment:
should this header be `X-Did` or similar throught? (or X-Did... something that's not so easily conflated with the word `did` 🤷). have a custom header called `Did` seems unusual

Reply:
X-Did is probably better, will change.